cluster/gce/util.sh

```diff
@@ -520,12 +520,14 @@ function tars_from_version() {
     SERVER_BINARY_TAR_URL="https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-server-linux-amd64.tar.gz"
     # TODO: Clean this up.
     KUBE_MANIFESTS_TAR_URL="${SERVER_BINARY_TAR_URL/server-linux-amd64/manifests}"
-    KUBE_MANIFESTS_TAR_HASH=$(curl ${KUBE_MANIFESTS_TAR_URL} --silent --show-error | ${sha1sum} | awk '{print $1}')
+    KUBE_MANIFESTS_TAR_HASH=$(curl "${KUBE_MANIFESTS_TAR_URL}" --silent --show-error | ${sha1sum})
+    KUBE_MANIFESTS_TAR_HASH=${KUBE_MANIFESTS_TAR_HASH%%[[:blank:]]*}
```

Comment:
I would like to hear from other cluster/ approvers on this.  I personally find this harder to comprehend than the awk version

Reply:
I guess that depends on how accustomed one is to `awk`. :-)
The `%%` replacement is a basic POSIX standard parameter expansion modifier, also clearly documented in the man pages.
And using shell internals makes the script more robust and portable (and efficient, even if that is not considered as needed).